src/content/docs/en/guides/backend/turso.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Turso & Astro
+description: Build locally with a SQLite file and deploy globally using Turso.
+type: backend
+service: Turso
+stub: false
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Turso](https://turso.tech) is a distributed database built on libSQL, a fork of SQLite. It is optimized for low query latency, making it suitable for global applications.
+
+### Prerequisites
```

Comment:
"```suggestion
## Initializing Turso in Astro

### Prerequisites
```
I realized I never did come back to put a proper level 2 heading here. Happy with "Initializing" since it matches the other guides. Would also be OK with "Setting up" or "Configuring" if you prefer!